### DISCUSSION TITLE
Gitlab Integration

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#5456] [New features]
Many teams use GitLab - we've had ~9 requests to date on this. 

User requests: #3971

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/15612601) 🔒 said:

> We would love it if you had a Gitlab integration, we don't use Jira

**Comment by jordanh** [comment]
This is love on production now!
